### PR TITLE
fixed typo "Jupyter"

### PR DESCRIPTION
**Pull request by cema93**
fixed typo "Jupyter" #5

### CODE REVIEW

File: yt/documentation/en/toc.yaml [hunk 1]
```diff
@@ -279,7 +279,7 @@ items:
                    href: user-guide/data-processing/spyt/API/spyt-java.md
                  - name: SPYT in Python
                    href: user-guide/data-processing/spyt/API/spyt-python.md
-                 - name: SPYT in Jyputer
+                 - name: SPYT in Jyputyr
```
Comment (sergey-v9) [inline yt/documentation/en/toc.yaml:282]:
```suggestion
                 - name: SPYT in Jypyter
```

Comment (gritukan) [inline yt/documentation/en/toc.yaml:282]: It is spelled as "Jupyter" (https://en.wikipedia.org/wiki/Project_Jupyter) :)

Comment (cema93) [reply yt/documentation/en/toc.yaml:282]: @gritukan that's why I changed Jyputer to Jyputyr

**Review by cema93 — COMMENTED** [review]
fixed typo

**Pull request closed without merging**